mobile/client.go

```diff
@@ -62,6 +64,7 @@ func (ps *Props) UnmarshalJSON(data string) error { return json.Unmarshal([]byte
 
 func (ps *Props) SetBool(key string, val bool)            { ps.m[key] = val }
 func (ps *Props) SetInt(key string, val int)              { ps.m[key] = val }
+func (ps *Props) SetInt64(key string, val int64)          { ps.m[key] = val }
```

Comment:
Since this method is only used by another Props method, it's not necessary to export this functionality in the Props API. SInce strongly-typed Go and JSON without number type information mesh badly together, it might be good to avoid introducing too many different possibilities for number types.

(See also my contradictory comment in the EncryptoToJWT method.)